posydon/unit_tests/utils/test_common_functions.py

```diff
@@ -0,0 +1,2271 @@
+            assert v in totest.ALL_STAR_STATES, "missing entry"
+        # check required STATE_UNDETERMINED
+        assert totest.STATE_UNDETERMINED in totest.ALL_STAR_STATES,\
+               "missing entry"
+        for v1 in totest.RICHNESS_STATES:
+            for v2 in totest.BURNING_STATES:
+                # check required values of combinatios of RICHNESS_STATES and
+                # BURNING_STATES
+                v = v1+"_"+v2
+                assert v in totest.ALL_STAR_STATES, "missing entry"
+
+    def test_value_MT_CASE_NO_RLO(self):
+        assert totest.MT_CASE_NO_RLO == 0
+
+    def test_value_MT_CASE_A(self):
+        assert totest.MT_CASE_A == 1
+
+    def test_value_MT_CASE_B(self):
+        assert totest.MT_CASE_B == 2
+
+    def test_value_MT_CASE_C(self):
+        assert totest.MT_CASE_C == 3
+
+    def test_value_MT_CASE_BA(self):
+        assert totest.MT_CASE_BA == 4
+
+    def test_value_MT_CASE_BB(self):
+        assert totest.MT_CASE_BB == 5
+
+    def test_value_MT_CASE_BC(self):
+        assert totest.MT_CASE_BC == 6
+
+    def test_value_MT_CASE_NONBURNING(self):
+        assert totest.MT_CASE_NONBURNING == 8
+
+    def test_value_MT_CASE_UNDETERMINED(self):
+        assert totest.MT_CASE_UNDETERMINED == 9
+
+    def test_value_ALL_RLO_CASES(self):
+        for v in [totest.MT_CASE_A, totest.MT_CASE_B, totest.MT_CASE_C,\
+                  totest.MT_CASE_BA, totest.MT_CASE_BB, totest.MT_CASE_BC,\
+                  totest.MT_CASE_NONBURNING]:
+            # check required values
+            assert v in totest.ALL_RLO_CASES, "missing entry"
+
+    def test_value_MT_CASE_TO_STR(self):
+        for v in [totest.MT_CASE_NO_RLO, totest.MT_CASE_A, totest.MT_CASE_B,\
+                  totest.MT_CASE_C, totest.MT_CASE_BA, totest.MT_CASE_BB,\
+                  totest.MT_CASE_BC, totest.MT_CASE_NONBURNING,\
+                  totest.MT_CASE_UNDETERMINED]:
+            # check required values
+            assert v in totest.MT_CASE_TO_STR.keys(), "missing entry"
+        for k,v in totest.MT_CASE_TO_STR.items():
+            assert isinstance(v, str)
+
+    def test_value_MT_STR_TO_CASE(self):
+        for k,v in totest.MT_STR_TO_CASE.items():
+            # check required items
+            assert v in totest.MT_CASE_TO_STR.keys()
+
+    def test_value_DEFAULT_CE_OPTION_FOR_LAMBDA(self):
+        assert totest.DEFAULT_CE_OPTION_FOR_LAMBDA == "lambda_from_profile_"+\
+               "gravitational_plus_internal_minus_recombination"
+
+
+class TestFunctions:
+    @fixture
+    def failing_csv_path(self, tmp_path):
+        # a temporary path to csv file for testing
+        path = totest.os.path.join(tmp_path, "hist_fail.csv")
+        with open(path, "w") as test_file:
+            test_file.write("0.0,1.0\n")
+            test_file.write("1.0,1.0\n\n")
+            test_file.write("2.0,1.0\n")
+        return path
+
+    @fixture
+    def failing_csv_path2(self, tmp_path):
```

Comment:
may want to give these failing_csv_paths a brief descriptive suffix instead of numbering them

Reply:
I used numbers, when I didn't had a short and descriptive name in mind.
Feel free to change the names to give all relevant information without getting too long names causing more line breaks.